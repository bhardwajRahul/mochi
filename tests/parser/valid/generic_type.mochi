let nums: list<int> = [1, 2]
