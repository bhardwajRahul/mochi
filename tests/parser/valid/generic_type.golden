(program
  (let nums
    (type list (type int))
    (list (int 1) (int 2))
  )
)
